<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>201</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>                  Сапёр</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>62</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>Начать игру</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OptionsButton">
       <property name="text">
        <string>Настройки</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>185</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Size_Layout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="Height_Indicator_label">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Высота</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLCDNumber" name="Height_lcdNumber"/>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="Width_lcdNumber"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Width_Indicator_label">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ширина</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="Size_label">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>РАЗМЕРЫ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Bombs_Layout">
     <item>
      <widget class="QLabel" name="Bomb_Indicator_label">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>БОМБ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="Bombs_lcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="DifficultLayout">
     <item>
      <widget class="QPushButton" name="Difficult_1">
       <property name="text">
        <string>Лёгкая ()</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Difficult_2">
       <property name="text">
        <string>Средняя ()</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Difficult_3">
       <property name="text">
        <string>Высокая ()</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>769</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
